### PR TITLE
[retirement_filters] add verra link to project details card

### LINKED ISSUE #863: [retirement] Improve UX of Verra project page link
This feature request came from a community member and was captured in #851 

**Feedback/Suggestion:** It's not obvious that to go to the corresponding Verra page for the project, you need to click the VCS code of the project. "Project Details Page" or something could be a band aid solution

**Developer Comment:** Agree with this. The verra project page link wasn't part of the initial brief but I included it because it felt sensible to do so. I think we can more intentionally expose the link.

**Next steps:**

- [ ] @theprofessora to provide design recommendation to improve the UX of this feature

### PR DESCRIPTION
**Pull request by 0xAeterno**
## Description

@theprofessora, I've made these further changes:
- "View project details" looked too wordy, "Project details" as an outbound link can be inferred from the iconography
- Made the font colour lighter to lower its visual heirarchy

## Related Ticket

Resolves #863 

## Changes

| Before  | After  |
|---------|--------|
|image|image|

## Checklist

- [x] I have run `npm run build-all` without errors
- [x] I have formatted JS and TS files with `npm run format-all`

### CODE REVIEW

**Review by Atmosfearful — APPROVED** [review]
Thanks! I look forward to the bug report when we finally add non-verra tonnes to our ecosystem and forget to update this file :)

**Comment by jabby09** [comment]
@Atmosfearful this change is showing up in staging but for some reason it has not shown up yet in production?

**Comment by 0xAeterno** [comment]
@jabby09 looks like this merge missed Thursday's automated release to production. Next release is Monday I think?
